subjects/gaming/stealth-boom/README.md

```diff
@@ -1,82 +1,98 @@
-
-- All AI should be implemented using Behavior Trees except for the Camera.
-
-- The player mission is up to you, either it can be some task to fix, kill all guards without getting caught or collect documents... Whatever you choose, the player should have enemies on his way to divert him away from his objective.
+  - Change the mouse sensitivity.
+  - Invert the mouse vertical axis.
+
+- Map/Level.
+
+  - This map should contain places for the player to hide from enemies by `Ducking`, `Hiding behind walls and props`.
+  - Pickable ammunition around the map.
+  - Pickable Health around the map.
+
+- Player:
 
+  - Walk and run animations.
+  - Shoot animation.
+  - Duck animation.
+  - Melee attack animation.
+  - Blood particles when hit.
+  - A health bar that should go down whenever the player gets damaged. When the player dies, he has the choice to either quit the game, go back to the main menu or start over.
+  - The player mission is up to you, it can be some task to fix, kill all enemies without getting caught or collect documents... Whatever you choose, the player should have enemies on his way to divert him away from his objective.
   - When the player successfully completes his mission, a pop up should appear saying that the mission is completed.
 
-- The player has a health bar that should go down whenever the player gets shot. When the player dies, he has the choice to either quit the game, go back to the main menu or start over.
+- Gun mechanics:
+
+  - a widget displaying the number of bullets.
+  - when the number of bullets is 0 the player is unable to shoot.
+  - shooting should have a sound effect and visual effect.
+  - Bullets visual impact on walls.
+
+- Enemy:
+
+  - Your game should contain atleast 2 enemy types `Melee` and `Ranged`.
+  - enemy AI should be made with behavior trees.
+  - enemy should be able to patrol around the map in a predefined path.
+  - enemy detects the player, if the player crosses its field of view.
+  - When the player enters the field of view of an enemy, the enemy enters into chasing mode and must start running after the player.
+    - `Ranged` enemies should take cover and shoot at the player.
+    - `melee` enemies should run close to the player and perform melee attacks.
+  - Enemies in chase mode alert nearby enemies making them enter chase mode as well.
+  - If the player leaves the field of view of all enemies for a certain time, the enemies go back to patrol mode.
+  - Enemies should have sounds effect whenever they change from chase to patrol mode, as well as the other way around.
+  - Enemies should have a visual indicator showing whether they are in patrol or chase mode
+
+- When pressing `Esc` the game is set on paused and a widget similar to the main menu widget pops up.
+- The player should be able to change the game graphics setting exactly like in the main menu.
+- A game should not last longer than 6 minutes. After that the same choices should appear as when the player dies.
```

Comment:
This contradicts the question in the audit:

`###### Is the lifespan of the game at least 6 minutes long from launch to mission completed?`